plugin/src/init.server.lua

```diff
@@ -3,12 +3,8 @@ local HttpService = game:GetService("HttpService")
 assert(plugin, "This code must run inside of a plugin")
 
 local toolbar = plugin:CreateToolbar("Luau") :: PluginToolbar
-local button = toolbar:CreateButton(
-	"Language Server Setup",
-	"Connect to Server",
-	"rbxassetid://10913079454",
-	"Connect to Server"
-) :: PluginToolbarButton
+local button =
+	toolbar:CreateButton("Language Server Setup", "Toggle Menu", "rbxassetid://11115506617", "Luau LSP") :: PluginToolbarButton
```

Comment:
```suggestion
	toolbar:CreateButton("Language Server Setup", "Toggle Menu", "rbxassetid://11115506617", "Luau Language Server") :: PluginToolbarButton
```

Can you change references of LSP to Language Server (I made a mistake of "Luau LSP" as it is technically wrong, but it's difficult to change existing references elsewhere)